submissions/hadizaSaadu/HTML/E-commerce.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>product-page</title>
+  </head>
+  <body bgcolor="#e6ac00">
+    <h1><center>DEEZAH'S CLOSET</center></h1>
+    <h3>
+      <i><center>Home of modest wears . . .</center></i>
+    </h3>
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+    <a href="#">HOME</a>
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+    <a href="#">ABOUT US</a
+    >&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+    <a href="#">CONTACT</a
+    >&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+    <a href="#">FAQ</a
+    >&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+    <hr />
+    <!--THIS SECTION IS FOR PRODUCTS-->
+    <section>
+      <h3>PRODUCTS SECTION</h3>
+      <table>
+        <tr>
+          <td>
+            <img
+              src="products/exclusive abaya.jpg"
+              ,width="300"
+              ,
+              height="450"
+            />
+
+            <p><strong>Comfy Exclusive Abaya</strong></p>
+            <p><strong>Price: #15,000</strong></p>
+            <p>
+              <strong>Product Description</strong><br />
+              this piece has been carefully designed to suit the need of all
+              modesty lovers,<br />it is a free size and fits for all
+              occassions, <br />
+              you would be missing out a great deal if you do not have this in
+              your closet!!!.
+            </p>
+          </td>
+          <td>
+            <img
+              src="products/umbrella style.jpg"
+              ,
+              width="300"
+              ,
+              height="450"
+            />
+
+            <p><strong>Comfy Umbrella Abaya</strong></p>
+            <p><strong>Price: #10,000</strong></p>
+            <p>
+              <strong>Product Description</strong><br />
+              this piece has been carefully designed to suit the need of all
+              modesty lovers,<br />it is a free size and fits for all
+              occassions, <br />
+              you would be missing out a great deal if you do not have this in
+              your closet!!!.
+            </p>
+          </td>
+        </tr>
+      </table>
+    </section>
+    <hr />
+    <h3>CHECK OUT OTHER AVAILABLE PRODUCTS BELOW:</h3>
+    <ol>
+      <li><a href="#">Casual Abaya</a></li>
+      <li><a href="#">Egyptian Model</a></li>
+      <li><a href="#">Nursing Abaya</a></li>
+    </ol>
+    <hr />
+    <footer>
+      <h3>CONTACTS US</h3>
+
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      <a href="https://www.facebook.com">
+        <img src="images/SM-logos/Facebook.png" , width="30" , height="40" />
```

Comment:
@bint-saad <img> does not need back slash. Remove it in all places it is used.

Reply:
Remove the `,` here too

Reply:
Done!!!